docs/resources/monitor.md

```diff
@@ -44,6 +44,8 @@ resource "datadog_monitor" "foo" {
 }
 ```
 
+**Note:** APM Trace Metrics are now available as distributions and your existing monitors have been migrated.  To learn more, or to see examples of how to update your terraform definitions to utilize the new distribution metrics, see the [migration guide](https://docs.datadoghq.com/tracing/guide/ddsketch_trace_metrics/)
```

Comment:
Some light suggestions: "APM latency data is now available as Distribution Metrics. Existing monitors have been migrated automatically but all terraformed monitors can still use the existing metrics. To learn more, or to see examples of how to update your terraform definitions to utilize the new distribution metrics, see the [detailed doc]"

Reply:
Can we say something like

> Terraformed monitors can still utilize the existing metrics, but we strongly recommend updating monitor definitions to query the new metrics. To learn more...